<launch>
    <arg name="yaml_file" default="/code/catkin_ws/src/user_code/quack-norris/params/apriltags.yaml" />
    <arg name="maps_yaml_file" default="/code/catkin_ws/src/user_code/quack-norris/params/maps.yaml" />
    <arg name="map_name" default="lap_small" />

    <node pkg="quack_norris" type="viz_node.py" name="viz_node" output="screen">
        <param name="yaml_file" value="$(arg yaml_file)" />
        <param name="maps_yaml_file" value="$(arg maps_yaml_file)" />
        <param name="map_name" value="$(arg map_name)" />
    </node>
</launch>